plugins/aws/flytekitplugins/aws/hpo.py

```diff
@@ -4,6 +4,7 @@
 
 from flyteidl.plugins.sagemaker import hyperparameter_tuning_job_pb2 as _pb2_hpo_job
 from flyteidl.plugins.sagemaker import parameter_ranges_pb2 as _pb2_params
+from flytekitplugins.aws.training import SagemakerBuiltinAlgorithmsTask, SagemakerCustomTrainingTask
```

Comment:
I did not mean rename `sagemaker` -> `aws`, I meant move `sagemaker` -> `aws/sagemaker`